circuit_definitions/README.md

```diff
@@ -0,0 +1,55 @@
+# Circuits definitions crate
+
+This crate contains the 'outer layer' for multiple circuits. The concrete circuits code is in `era-zkevm_circuits`
+repository.
+
+## Code structure
+
+We have 13 different 'base layer' circuits (for example MainVM, Decomitter), and 3 recursive circuits (Leaf, Node and
+Scheduler).
+
+Base layer circuits are located in `src/base_layer`, Recursive circuits are in `src/recursion_layer`.
+
+We also have 2 'AUX' circuits: compressor and wrapper, that are run on top of the final Scheduler proof, and they are
+located in `src/aux_layer`.
+
+![circuits](https://user-images.githubusercontent.com/128217157/275817097-0a543476-52e5-437b-a7d3-10603d5833fa.png)
```

Comment:
I am not sure that there is a limitation on number of nodes for circuit type. The recursive tree should be able to be any size.

Reply:
PS, I think you mean 1 node circuit goes to scheduler. Sorry for misunderstanding.